modules/internal/LocaleTree.chpl

```diff
@@ -27,25 +27,31 @@ module LocaleTree {
   use ChapelLocale; // For declaration of rootLocale.
 
   record chpl_localeTreeRecord {
-    var left, right: locale?;
+    var left, right: locale;
+
+    proc init() {
+    }
```

Comment:
Is this empty initializer required?  (my sense is that it will have the same effect as the default initializer would?)